subsys/partition_manager/pm.yml.pgps

```diff
@@ -3,10 +3,14 @@
 pgps:
   placement:
     before: [tfm_storage, end]
+#ifdef CONFIG_PM_PARTITION_REGION_PGPS_EXTERNAL
+  region: external_flash
+#else
```

Comment:
Shouldn't the `inside: [nonsecure_storage]` be inside this `#else` clause, so that we only try to place the `pgps` partition inside the non-secure storage if its in internal flash? The secure partition manager (TF-M to be precise) has no control over the security of the external flash (only the configuration of the peripherals used to hook it up).

Reply:
This could be fixed with the suggestion in my other comment.

Reply:
agreed -- will do both tomorrow